requirements.txt

```diff
@@ -119,7 +117,7 @@ typing-inspect==0.8.0
 typing_extensions==4.5.0
 tzdata==2023.3
 tzlocal==4.3
-urllib3==2.0.2
+urllib3==1.26.6

```

Comment:
Same with urllib. I had an issue with the the new 2.0.2 version of the library and the SSL handling when calling the OpenAI API. I decided to go here with an earlier version of the library.